lib/rubycritic/configuration.rb

```diff
@@ -10,10 +10,11 @@ class Configuration
                   :feature_branch, :base_branch_score, :feature_branch_score,
                   :base_root_directory, :feature_root_directory,
                   :compare_root_directory, :threshold_score, :base_branch_collection,
-                  :feature_branch_collection, :churn_after, :ruby_extensions
+                  :feature_branch_collection, :churn_after, :ruby_extensions, :paths
 
     def set(options)
       self.mode = options[:mode] || :default
+      self.paths = options[:paths] || ['.']
```

Comment:
@Fito Maybe this should [default to this](https://github.com/whitesmith/rubycritic/blob/cafe4c92f4a49cede49ab6cb3231b1af651c7466/lib/rubycritic/cli/options/file.rb#L58-L60)?

Reply:
The `path` (singular) option currently refers to the path where the reports will get written. The `paths` (plural) option refers to the paths to be analyzed by RubyCritic. There might be an opportunity to rename either of those concepts for clarity, but it feels outside the scope of this change. The names were already in the codebase.